spec/repository_spec.rb

```diff
@@ -26,3 +26,17 @@
     expect(EXTRA_FILES).to eq([])
   end
 end
+
+describe "The file \"renamed-styles.json\"" do
+  it "must be valid JSON" do
+    expect(JSON_IS_VALID).to be true
+  end
```

Comment:
Turns out that `let(:style_validates) { CSL.validate(path).length.zero? }` is run for every example that calls `style_validates`, which resulted in style validation being done twice. With `before(:all)` it seems we just saved about 20 seconds: https://github.com/citation-style-language/styles/commit/1e93b01700c7a05155ec68b5cb6e474e11a8f499

We're under 1 minute of test time, now:

```
Finished in 57.81 seconds (files took 45.74 seconds to load)
132028 examples, 0 failures
```